apps/src/lib/kits/maker/boards/microBit/LightSensor.js

```diff
@@ -10,8 +15,19 @@ export default class LightSensor extends EventEmitter {
       rangeMax: 255,
       currentReading: 0
     };
+    // Track 3 seconds of historical data over which we can average sensor readings
+    this.buffer = new Float32Array(MAX_SENSOR_BUFFER / SAMPLE_RATE);
+    this.bufferIndex = 0;
```

Comment:
Maybe `currentBufferWriteIndex` just to be really explicit about what this value is and what it's tracking?